### PR TITLE
NIFI-4112: Fix ConvertExcelToCSV to handle empty cells.

### PR DESCRIPTION
**Pull request by ijokarumawak**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [x] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/with-blank-cells.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+A,B,C,D
```
Comment (m-hogue) [inline nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/with-blank-cells.csv]: Would a test case with all empty cells be useful as well? Thinking edge cases.

**Comment by m-hogue** [comment]
Hey @ijokarumawak : LGTM. Just had a single comment.

File: nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/with-blank-cells.csv (hunk 1 above)
Comment (ijokarumawak) [inline nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/with-blank-cells.csv]: @m-hogue Thanks for your comment! If a row only contains empty cells, then it will be ignored. Since we're processing an Excel sheet as an XML document, such row wouldn't be passed to our code. Excel XML only contains rows and cells those have values.

**Comment by mattyb149** [comment]
@m-hogue Thank you for the review and feedback! this LGTM to me too, ran tests with various number of missing columns. Thanks! Merging to master
